sdk/monitor/azquery/README.md

```diff
@@ -117,66 +120,104 @@ The timespan can be the following string formats:
   - [Metrics result structure](#metrics-result-structure)
 
 ### Logs query
+The example below shows a basic logs query using the `QueryWorkspace` method. `QueryWorkspace` takes in a [context][context], a [Log Analytics Workspace][log_analytics_workspace] ID string, a [Body](#logs-query-body-structure) struct, and a [LogsClientQueryWorkspaceOptions](#increase-wait-time-include-statistics-include-render-visualization) struct and returns a [Results](#logs-query-result-structure) struct.
+
 ```go
-client := azquery.NewLogsClient(cred, nil)
-timespan := "2022-08-30/2022-08-31"
+import (
```

Comment:
For full, end-to-end examples, we're moving away from putting them in markdown as they don't compile and it's too easy to end up with various copy/paste mistakes.

Take a look at some of the migration examples in [`azeventhubs`](https://github.com/Azure/azure-sdk-for-go/tree/main/sdk/messaging/azeventhubs) and how they link to example files.

Reply:
fixed in commit [d5e54b6](https://github.com/Azure/azure-sdk-for-go/pull/19208/commits/d5e54b6debc04da260e9cf6b8205c4565b57c56a)